python/test/state_representation/test_trajectory.py

```diff
@@ -0,0 +1,276 @@
+import unittest
+import datetime
+from random import shuffle
+
+from state_representation import (
+    CartesianState,
+    JointState,
+    CartesianTrajectory,
+    JointTrajectory,
+    StateType
+)
+
+from state_representation.exceptions import (
+    EmptyStateError,
+    IncompatibleStatesError,
+    IncompatibleReferenceFramesError,
+    IncompatibleSizeError,
+)
+
+
+class TestState(unittest.TestCase):
+    __num_random_tests = 10
+
+    __cartesian_state_attributes = [
```

Comment:
That's maybe a bit over the top, since we could just compare the `data()` vector but why not :smile:

Reply:
true, just didn't want to hardcode too many numbers in the code. 10 is a bit too much :smile: